File: AndroidBannerSliderExample/app/src/main/java/com/example/shakil/androidbannersliderexample/Interface/INaturalBeautyLoadListener.java
package com.example.shakil.androidbannersliderexample.Interface;

import com.example.shakil.androidbannersliderexample.Model.Banner;
import com.example.shakil.androidbannersliderexample.Model.NaturalBeauty;

import java.util.List;

public interface INaturalBeautyLoadListener {
    void onNaturalBeautyLoadSuccess(List<NaturalBeauty> naturalBeauties);
    void onNaturalBeautyLoadFailed(String message);
}

File: KotlinBannerSlider/app/src/main/java/com/example/shakil/kotlinbannerslider/Interface/INaturalBeautyLoadListener.kt
package com.example.shakil.kotlinbannerslider.Interface

import com.example.shakil.kotlinbannerslider.Model.NaturalBeauty

interface INaturalBeautyLoadListener {
    fun onNaturalBeautyLoadSuccess(naturalBeauties: List<NaturalBeauty>)
    fun onNaturalBeautyLoadFailed(message: String)
}